config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/lambda-controller
-  newTag: v0.1.3
+  newTag: v0.1.2
```

Comment:
This should stay `v0.1.3`, make sure to run `git fetch --tags --all` in lambda-controller repo before regenerating the code

Reply:
Done